src/vpn-worker.ts

```diff
@@ -182,6 +177,11 @@ const worker = async (instanceId: number, serviceId: number) => {
 		verbose,
 	);
 
+	vpn.on('process:uncaughtException', async (err, origin) => {
```

Comment:
@Page- this is superfluous also?

Reply:
Yes, and the correct way to catch it would actually be
```suggestion
	process.on('uncaughtException', async (err, origin) => {
```
see: https://nodejs.org/docs/latest-v16.x/api/process.html#event-uncaughtexception

The `process:error`/`process:exit` errors the `VpnManager` interface has are for the child_process interface to the openvpn process, whereas the `uncaughtException` is on the nodejs process